aspnetcore/signalr/get-started-signalr-core/sample/Hubs/ChatHub.cs

```diff
@@ -9,7 +9,7 @@ public class ChatHub : Hub
         public Task Send(string user, string message)
         {
             string timestamp = DateTime.Now.ToShortTimeString();
-            return Clients.All.SendAsync(timestamp, user, message);
+            return Clients.All.SendAsync(nameof(Send), timestamp, user, message);
```

Comment:
This should not be `nameof`. The name expected here is the name of the **client** method, which is not guaranteed to be the same as the server method. We shouldn't set the assumption that they will be the same. This is why I keep advocating for these samples to use "Receive" as the name on the client but @davidfowl doesn't like that for some reason :)

Reply:
I second the idea of using a different name. As you can see, I was under the impression the names were to be the same.